SIA.tex

```diff
@@ -52,45 +45,22 @@ \section{Introduction}
-    \item SIAV2  supports the new DataLink  standard, providing additional capabilities for data access as well as the ability to link to related data products, as well as the VO Image data model standard currently under development, providing a full description of the structure of an image dataset.
-\end{itemize}
-While some of the more advanced capabilities for dynamic access to multi-dimensional images are still under development, the initial specification provides the capability to discover and download multi-dimensional image datasets via a service interface consistent with current VO standards
-
-\subsection{Current and Future support}
- These are the list of changes which  will be considered in a later version of this protocol  (SIA-2.1):
-\begin{itemize}
-    \item The protocol should support  a detailed {metadata} capability. In the current design, this   capability will take a single parameter with an identifier (typically a publisher dataset identifier from a data discovery response) and return a document with detailed metadata about the dataset. The primary usage is to drill-down to the detailed  metadata after discovery
-    \item The standardID for the {metadata} capability will  probably be : ivo://ivoa.net/std/SIA\#metadata-2.1  
-    \item  The parameter language should be extended to support case-insensitivity, wildcards, and pattern recognition.
-    \item The DALI UPLOAD facility will be defined in order to allow queries based on values stored in a file.
-    \item The protocol will also support an option for virtual data discovery. With this option, controlled by usage of an optional parameter, the  query can force the discovery of virtual datasets with best  matching to the input parameter constraints
-    \item COLLECTION and FACILITY currently provide query parameters provide selection on service defined set of strings. It would be good to define standardized lists of those at IVOA level for next version.
-    \item Visibility data and event lists are not considered as valid DPTYPES in this version due to complexity of data access functionalities for those dataset types.  
-\end{itemize}
+DsSAP specifies standardID values for each capability, as defined by VODataService \citep{std:VODS11}. DsSAP services may be registered in an IVOA Registry using the SimpleDALRegExt \citep{std:DALREGEXT} extension schema.
+\subsection{Changes from SIA-2.0i to DsSAP}
 
+Virtual Observatory access to astronomical images has been available via the SIA-1.0 protocol for over a decade.  Many such services have been implemented since 2002, and SIA-1.0 \citep{std:SIAP} was formally standardized as an IVOA Recommendation in 2009. SSA \citep{std:SSA} played a similar role for spectra and also used specific metadata in the query response. SIA-2.0 \citep{std:SIA2} was multi-dimensional and fully integrated with the modern VO architecture and related standards, but restricted to images and data cubes.
```

Comment:
long lines like this should be broken up to assist readability of the code. This applies to quite a bit of the new content.

Reply:
OK